<launch>
1;3201;0c  <node pkg="point_cloud_scene_decomposer" type="pcl_filter_utils_node" 
	name ="pcl_filter_utils" output="screen">
    <remap from="input" to="/camera/depth_registered/points"/>
    <remap from="target" to="pcl_filter_utils/output/cloud" />
  </node>
  <node pkg="rqt_reconfigure" type="rqt_reconfigure"
	name="detector_reconfigure" />
</launch>
